lib/Cake/Test/Case/Utility/CakeTextTest.php

```diff
@@ -16,18 +16,18 @@
  * @license       http://www.opensource.org/licenses/mit-license.php MIT License
  */
 
-App::uses('String', 'Utility');
+App::uses('CakeText', 'Utility');
```

Comment:
Yes. That would be quite the BC break. It also lines up with CakeNumber and CakeTime then.